app/src/main/java/com/github/onlynotesswent/ui/overview/FolderContent.kt

```diff
@@ -93,14 +94,17 @@ fun FolderContentScreen(
 
   var updatedName by remember { mutableStateOf(folder.value!!.name) }
   var expandedFolder by remember { mutableStateOf(false) }
-
+  var showFileSystemPopup by remember {
+    mutableStateOf(true)
```

Comment:
Shouldnt the default value of showFileSystemPopUp be false?

Reply:
Yes, I actually scrapped the button from the FolderContent as for now our button only is used temporarily to see my features work but the actual place the popup should be displayed is from the drop down menus of notes and an option for folders which will be done later.